mehive-server/dbconfig.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
Are both of these dbconfig files required? Seems a bit redundant to have the same file in two spots.